tests/test_schemas.py

```diff
@@ -194,3 +195,14 @@ def build_rule(query):
             build_rule("""
                     process where process.pid == "some string field"
             """)
+
+
+class TestVersions(unittest.TestCase):
+    """Test that schema versioning aligns."""
+
+    def test_stack_schema_map(self):
+        """Test to ensure that an entry exists in the stack-schema-map for the current package version."""
+        package_version = load_current_package_version()
+        stack_map = utils.load_etc_dump('stack-schema-map.yaml')
+        err_msg = f'There is no entry defined for the current package ({package_version}) in the stack-schema-map'
+        self.assertIn(package_version, ['.'.join(v.split('.')[:2]) for v in stack_map], err_msg)
```

Comment:
this is a little cryptic. maybe... this?
```suggestion
        self.assertIn(package_version, [str(Version(v)[:2]) for v in stack_map], err_msg)
```